source/MRMesh/MRMeshRelax.cpp

```diff
@@ -59,6 +66,18 @@ void relaxKeepVolume( Mesh& mesh, const RelaxParams params )
     std::vector<Vector3f> vertPushForces( zone.size() );
     for ( int i = 0; i < params.iterations; ++i )
     {
+        SimpleProgressCallback internalCb1, internalCb2;
+        if ( cb )
+        {
+            internalCb1 = [&] ( float p )
+            {
+                cb( ( float( i ) + p * 0.5f ) / float( params.iterations ) );
```

Comment:
`+ p * 0.5f` to be added after  `/ float( params.iterations )` ?

Reply:
this seems correct